pallets/messages/src/benchmarking.rs

```diff
@@ -35,6 +35,20 @@ benchmarks! {
 		}
 	}: _ (RawOrigin::Signed(caller), None, 1, input)
 
+	add_ipfs_message {
+		let n in 0 .. T::MaxMessagePayloadSizeBytes::get() - 1;
+		let m in 1 .. MESSAGES;
+		let caller: T::AccountId = whitelisted_caller();
+		let input = vec![1; n as usize];
+		let cid = CIDv2::new(input);
+		let payload_length = 1_000;
+
+		// for j in 0 .. m {
```

Comment:
I think you should uncomment and use m to register messages to have a more accurate benchmark.